src/crawlee/storages/_key_value_store.py

```diff
@@ -15,6 +17,7 @@
     from crawlee.configuration import Configuration
 
 T = TypeVar('T')
+DictT = TypeVar('DictT', bound=dict[str, Any])
```

Comment:
Sorry, but what is this good for?

Reply:
Thank you for pointing that out to me.

A forgotten artefact that was left over from the implementation.